src/components/Checkout.js

```diff
@@ -1,5 +1,52 @@
-const Checkout = () => {
-  return <p>Checkout here</p>
+import Form from './Form'
+import { useNavigate } from 'react-router-dom'
+import '../styles/_Checkout.scss'
+
+const Checkout = ({shoppingBag, totalPrice}) => {
+
+  const navigate = useNavigate()
+
+  if(!shoppingBag.length) {
+    navigate('/shopping-bag')
+  }
+
+  const items = shoppingBag.map(item => {
+    return <div className='request-item' key={item.id}>
+      <div>
+        <b>{item.quantity}x {item.type}</b>
+        <p>Color: {item.color}</p>
+        {item.size !== 'onesize' && <p>Size: {item.size}</p>}
+      </div>
+      <b>${(item.quantity * item.price)}</b>
+    </div>
+  })
+
+  const numberOfItems =
+    shoppingBag.reduce((acc, curr) => {
+      return acc + curr.quantity
+    }, 0)
+
+  return (<div className="checkout">
+    <div className="checkout-header">
```

Comment:
Rachel and I have been using BEMS conventions for class naming which instead of using dashes, uses underscores. So instead of checkout-header, it'd be checkout__header. Either way works for me, but we should probably set a convention for uniformity.

Reply:
agreed. Let's discuss this in the standup